runtime/src/lib.rs

```diff
@@ -109,7 +109,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     // implementation changes and behavior does not, then leave spec_version as
     // is and increment impl_version.
     spec_version: 237,
-    impl_version: 0,
+    impl_version: 1,
```

Comment:
Arguably would increment here the `spec_version`. Since adding a new pallet, changes the metadata and supported types by clients.

Reply:
changed 👍